README.md

```diff
@@ -93,6 +93,10 @@ You can configure the version and properties adjustments for specific branches a
   - has to be a **full match pattern** e.g. `v(.+)`, default is `.*`
 - `<describeTagFirstParent>` Enable(`true`) or disable(`false`) following only the first parent in a merge commit
   - default is `true`
+- `<describeTagMaxDepth>` An integer that describes the maximum number of commits to scan in search of a tag matching
```

Comment:
This description is no longer accurate I need to fix it: it's the max distance with the ref commit and not a number of commits.